.github/workflows/python-test-package.yml

```diff
@@ -1,13 +1,9 @@
 # This workflow will install Python dependencies and run tests
 # For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
 
-name: Python Testing
+name: Run Tests
 
-on:
-  push:
-    branches: [ "main" ]
-  pull_request:
```

Comment:
Why not run tests on pull requests anymore?

Reply:
ik heb codecov criteria er uit gehaald. Buiten scope van dee story. zal een story toevoegen aan backlog voor ophogen coverage img2cat

Reply:
Probleem wat je krijgt is als derden een PR maken, deze pending blijven omdat de tests niet runnen. Branch protection rules kunnen ze niet laten draaien, maar gaan falen omdat ze niet slagen (dit is een probleem geweest in deze repo). Ik zou daarom heel sterk overwegen de `pull_request` trigger terug te plaatsen, desnoods zonder condition op de branch.

Reply:
additional aan pull request kan geen kwaad